src/parsers/pulldown.rs

```diff
@@ -476,7 +494,19 @@ impl CommonMarkViewerInternal {
                 newline(ui);
             }
             pulldown_cmark::Event::TaskListMarker(mut checkbox) => {
-                ui.add(Checkbox::without_text(&mut checkbox));
+                if options.mutable {
```

Comment:
Immutable uses original checkbox widget that you made, while mutable uses normal egui checkbox widget.